pom.xml

```diff
@@ -33,6 +39,93 @@
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-ui</artifactId>
+			<version>1.7.0</version>
+		</dependency>
+
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<version>1.5.21</version>
+		</dependency>
+
+		<!-- HTTP client: jersey-client -->
+		<dependency>
```

Comment:
There are a couple dependencies in this file that don't seem necessary, e.g. this one, migbase64.
Please remove them, just keep in what you need for now.